### PR TITLE
Add io.github.roseblume.rosary

### PR DESCRIPTION
**Pull request by RoseBlume**
### Please confirm your submission meets all the criteria
This application is a simple bible app designed with the goal of providing Christians with reliable sources to review so that they may come to their own conclusions about their faith.
- [X] Please describe your application briefly.
- [X] I have read the [App Requirements](https://docs.flathub.org/docs/for-app-authors/requirements) and [App Maintenance](https://docs.flathub.org/docs/for-app-authors/maintanance) pages.
- [X] My pull request follows the instructions at [App Submission](https://docs.flathub.org/docs/for-app-authors/submission).
- [X] I have [built](https://docs.flathub.org/docs/for-app-authors/submission/#before-submission) and tested the submission locally.
- [X] I am using only the minimal set of permissions. (If not, please explain each non-standard permission.)
- [X] All assets referenced in the manifest are redistributable by any party. If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. Link:https://github.com/RoseBlume/Rosary
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. (If not, explain why.)

### CODE REVIEW

**Comment by hfiguiere** [comment]
why did you close the other?

File: .flatpak-builder/ccache/disabled/ccache.conf
Comment (hfiguiere) [inline .flatpak-builder/ccache/disabled/ccache.conf:1]: remove this. this doesn't belong here.

Comment (RoseBlume) [reply .flatpak-builder/ccache/disabled/ccache.conf:1]: Fixed

**Comment by RoseBlume** [comment]
> why did you close the other?

**Comment by RoseBlume** [comment]
This version has been tested and will should work with the bot. It would be appreciated if you could run the bot.

**Comment by RoseBlume** [comment]
bot, build io,github.roseblume.rosary

**Comment by RoseBlume** [comment]
@hfiguiere It has passed all the bots checks, can you please confirm that it is ready?

**Comment by hfiguiere** [comment]
> @hfiguiere It has passed all the bots checks, can you please confirm that it is ready?

can you answer unanswered questions?

**Comment by RoseBlume** [comment]
@hfiguiere Errors were appearing when I was trying to push. For some reason it kept saying that it failed to upload everything.

**Comment by bbhtt** [comment]
Can you fix the checklist in the description? This is the second PR where everything is wrong there.

**Comment by RoseBlume** [comment]
> Can you fix the checklist in the description? This is the second PR where everything is wrong there.

@bbhtt Fixed

**Comment by RoseBlume** [comment]
Im just wondering but how is this merged?

**Comment by bbhtt** [comment]
Into a new repo for your app. You'll have to wait, we batch the merges.

**Comment by RoseBlume** [comment]
bot, build io.github.roseblume.rosary

**Pull request closed without merging**